tests/test_flows.py

```diff
@@ -244,23 +244,42 @@ def init(self):
         self.mac_repeater2_wlan2 = re.search(rb"link/ether " + RE_MAC, mac_repeater2_wlan2_output).group('mac').decode()
         self.debug("Repeater2 wl2: {}".format(self.mac_repeater2_wlan2))
 
+    def _check_log_internal(self, device: str, program: str, regex: str) -> bool:
+        '''Search for regex in logfile for program on device.'''
+        logfilename = os.path.join(self.rootdir, 'logs', device, 'beerocks_{}.log'.format(program))
+        with open(logfilename) as logfile:
+            for line in logfile.readlines():
+                if re.search(regex, line):
+                    self.debug("Found '{}'\n\tin {}".format(regex, logfilename))
+                    return True
+        return False
+
     def check_log(self, device: str, program: str, regex: str) -> bool:
         '''Verify that on "device" the logfile for "program" matches "regex", fail if not.'''
-        logfilename = os.path.join(self.rootdir, 'logs', device, 'beerocks_{}.log'.format(program))
         try:
-            # HACK check_log is often used immediately after triggering a message on the other side.
+            # HACK self.check_log(is, "often", rused immediately after triggering a message on the other side.)
```

Comment:
This seems to be a find/replace job gone wrong `s/check_log (\S+) (\S+) (.*)/self.check_log(\1, "\2", r\3)/g`